build.gradle

```diff
@@ -201,12 +201,13 @@ subprojects {
         if (System.getenv('PROTO_VERSION') != null) {
             protoMajorVersion = System.getenv('PROTO_VERSION')
         } else {
-            protoMajorVersion = '2'
+            protoMajorVersion = '3'
         }
         if (protoMajorVersion == "2") {
             protobufVersion = protobuf2Version
         } else if (protoMajorVersion == "3") {
             protobufVersion = protobuf3Version
+            setProperty('coreNotStrict', 'true')
```

Comment:
This is necessary so that if someone runs `./gradlew build`, they don't get validation errors from the proto3 generated code. I thought about removing the `coreNotStrict` flag entirely and replacing it with checks on the `protoMajorVersion`, but I figured this was less invasive, though I could be argued either way